### DISCUSSION TITLE
Considered regenerate the sample default config?

### DISCUSSION THREAD

**Discussion by llc0930** [ClementTsang/bottom#1592] [Q&A] [answered]
Have you considered regenerate regarding the [disk.mount_filter] section in the sample default config?
`.*` doesn't work.

And `#memory_legend = "TopRight"` and `#network_legend = "TopRight"` should be "top-right"

**Comment by ClementTsang** [comment]
Sure, I'll take a look at this.

**Reply by ClementTsang** [reply, marked as answer]
Hm, `.*` works fine if regex is enabled for me.

For example, given this:

![image](https://github.com/user-attachments/assets/5914c468-2021-477a-bf3a-716608b82b3e)

if I do:

```toml
[disk.mount_filter]
list = ["/mnt/.*"]
regex = true
```

it filters it out correctly:

![image](https://github.com/user-attachments/assets/1e76cbdc-34f6-4f27-b642-2d571bbfe65d)

**Reply by llc0930** [reply]
I recently noticed that my original filter list was not taking effect and the setting changed to `[disk.mount_filter]`, so I updated the configuration file.

So the default value is no longer `regex = true`?
Or is it just because I uncommented it before that the default value is always false?

**Reply by ClementTsang** [reply]
The default is false, yes. I suppose I can make a note of that in the default config.

**Reply by llc0930** [reply]
Anyway, I also took the opportunity to rewrite my filters as simple strings, maybe that would be good too.
